core/src/scene/stops.cpp

```diff
@@ -75,6 +75,61 @@ auto Stops::FontSize(const YAML::Node& _node) -> Stops {
     return stops;
 }
 
+auto Stops::Sizes(const YAML::Node& _node, const std::vector<Unit>& _units) -> Stops {
+    Stops stops;
+    if (!_node.IsSequence()) {
+        return stops;
+    }
+
+    for (auto& unit : _units) {
+        if (unit != Unit::pixel) {
```

Comment:
Those units come from the https://github.com/tangrams/tangram-es/blob/master/core/src/scene/styleParam.cpp#L93 for the specific style param key, not from the style parsed data, it seems that can't ever happen unless we change what's in the static variable.